.bazelrc

```diff
@@ -4,12 +4,18 @@
 # bazel configurations for running tests under sanitizers.
 # Based on https://github.com/bazelment/trunk/blob/master/tools/bazel.rc
 
+# TODO: Remove once support is added, avoid MODULE.bazel creation for now
+common --enable_bzlmod=false
+
 # Enable automatic configs based on platform
 common --enable_platform_specific_config
 
 # Needed by gRPC to build on some platforms.
 build --copt -DGRPC_BAZEL_BUILD
 
+# Workaround abseil libraries missing symbols
+build:windows --linkopt=/FORCE
+
```

Comment:
How is this a workaround ?

If I understand correctly:

https://learn.microsoft.com/en-us/cpp/build/reference/force-force-file-output?view=msvc-170#remarks

this will just crash at runtime.

Reply:
My understanding for our use case in tests here is that windows does not support no-deps cc libraries, which bazel is relying on, so we're skipping the issues at link time, which will never be satisfied, but then since they exist at runtime, the tests pass.

Reply:
Which test in particular is affected by this ?

If this is about the api singleton test (which is not executed in windows anyway),
I would rather remove this work around, and not build the singleton test under windows+bazel at all,
or even not under bazel at all, if this helps.

Reply:
it is about that one being _built_ on windows https://github.com/keith/opentelemetry-cpp/blob/9fedbc258e5e101d81ee889b8f7ad33574c5440c/api/test/singleton/BUILD, so yea I can disable building this on windows instead if that's what you'd prefer

Reply:
pushed!